state-chain/primitives/src/lib.rs

```diff
@@ -122,6 +131,13 @@ pub const MILLISECONDS_PER_BLOCK: u64 = 6000;
 
 pub const SECONDS_PER_BLOCK: u64 = MILLISECONDS_PER_BLOCK / 1000;
 
+/// Number of blocks to wait before trying a previously failed swap again
+pub const SWAP_RETRY_DELAY_BLOCKS: u32 = 5;
+
+/// Max allowed value for the number of blocks to keep retrying a swap before it is refunded
+pub const MAX_RETRY_DURATION_BLOCKS: u32 =
+	3600 / (SECONDS_PER_BLOCK as u32 * SWAP_RETRY_DELAY_BLOCKS);
```

Comment:
> Also, not sure what the significance of * SWAP_RETRY_DELAY_BLOCKS

The idea was to tie it to the number of times the swap can be retried. 

>  this constant probably doesn't need to be in the primitives

I think I moved it because it was used in tests inside cf-swapping, but it is not anymore so I will move it back.

> and we may want to configure it.

Sounds like this can be its own PR?